api/app/src/external/commonwell/document/document-query.ts

```diff
@@ -79,6 +79,12 @@ export async function queryAndProcessDocuments({
         MAPIWebhookStatus.completed,
         toDTO(documentsSandbox)
       );
+
+      await updateDocQuery({
+        patient: { id: patient.id, cxId: patient.cxId },
+        downloadProgress: { status: "completed" },
+      });
```

Comment:
Couldn't we just set it to `completed` [here](https://github.com/metriport/metriport/pull/572/files#diff-e8f35fbd0f42056807c42b35e27fe7bb22750b6b27273e82a57502b227f49bd7R76)?

If we heep this here, I'd suggest updating the DB before the WH update (line 75)